### PR TITLE
image and input component

### PR DESCRIPTION
**Pull request by rohanmathur91**
Please review the components.

### CODE REVIEW

File: input/input.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://utilities91.netlify.app/styles.css" />
+    <link rel="stylesheet" href="./input.css" />
```
Comment (smilegupta) [inline input/input.html]: In line 9 looks like you have imported css and i see you have imported it in polorid css also... is it intentional. If yes, please ignore my comment

File: input/input.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <div>
+        <label for="password">Name</label>
+        <input class="py-1 px-2 text-base border rounded-sm" id="password" type="password"
+            placeholder="Enter your password">
+        <span class="success-msg p-1 rounded-sm red">Login successfull!</span>
+    </div>
+
+    <div>
+        <label for="email">Name</label>
+        <input class="py-1 px-2 text-base border rounded-sm" id="email" type="email" placeholder="Enter your email">
```
Comment (smilegupta) [inline input/input.html]: label for email and you mentioned label content as Name.. is it intentional? If yes ignore the comment.

File: input/input.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+<body>
+    <div>
+        <label for="name">Name</label>
+        <input class="py-1 px-2 text-base border rounded-sm" id="name" type="text" placeholder="Enter your name">
+    </div>
+
+
+    <div>
+        <label for="password">Name</label>
```
Comment (smilegupta) [inline input/input.html]: label for password and you mentioned label content as Name.. is it intentional? If yes ignore the comment.

**Review by smilegupta — APPROVED** [review]
Looks good to me! There are 2-3 comments I have added, if its intentional ignore my comments. Else correct them

File: input/input.html (hunk 1 above)
Comment (rohanmathur91) [reply input/input.html]: Forgot to remove that file :)

**Review by smilegupta — APPROVED** [review]

**Review by anupkgurung — APPROVED** [review]
looks good

**Pull request merged**